minargon/templates/icarus/layout.html

```diff
@@ -94,13 +94,13 @@
                   aria-controls="navbarsExampleDefault" aria-expanded="false" aria-label="Toggle navigation">
             <span class="navbar-toggler-icon"></span>
           </button>-->
-          <div class="navbar-collapse collapse" id="navbar">
+          <div class="navbar-collapse collapse bg-maroon" id="navbar">
```

Comment:
why add the class?

Reply:
I will remove this, not necessary here, it slipped after debugging